fast_llm/tensor.py

```diff
@@ -369,3 +369,9 @@ def init_(meta: ParameterMeta, tensor: torch.Tensor, generator: torch.Generator)
             return tensor
 
     return init_
+
+
+def bias_init_method(conv_weight):
```

Comment:
Method is much more specific than the name suggest. Use `init_uniform_` directly instead?